doc/Manuals/ig-ss_x-road_v6_security_server_installation_guide_for_rhel7.md

```diff
@@ -0,0 +1,297 @@
+ 1.6  | TCP 4000                                  | User interface (local network)
+ 1.7  | TCP 8080                                  | Information system access points (in the local network)<br> Connections from information systems
+ &nbsp; | TCP 8443                                | Information system access points (in the local network)<br> Connections from information systems
+ 1.8  |                                           | Security server internal IP address(es) and hostname(s)
+ 1.9  |                                           | Security server public IP address, NAT address
+
+
+### 2.3 Requirements for the Security Server
+
+Minimum recommended hardware parameters:
+
+-   the server’s hardware (motherboard, CPU, network interface cards, storage system) must be supported by RHEL7 in general;
+
+-   a 64-bit dual-core Intel, AMD or compatible CPU; AES instruction set support is highly recommended;
+
+-   2 CPU;
+
+-   4 GB RAM;
+
+-   10 GB free disk space (OS partition), 20-40 GB free disk space (`/var` partition);
+
+-   a 100 Mbps network interface card;
+
+-   if necessary, interfaces for the use of hardware tokens.
+
+Requirements to software and settings:
+
+-   an installed and configured RHEL7 (v7.3 or newer) x86-64 operating system;
+
+-   if the security server is separated from other networks by a firewall and/or NAT, the necessary connections to and from the security server are allowed (**reference data: 1.4; 1.5; 1.6; 1.7**). The enabling of auxiliary services which are necessary for the functioning and management of the operating system (such as DNS, NTP, and SSH) stay outside the scope of this guide;
+
+-   if the security server has a private IP address, a corresponding NAT record must be created in the firewall (**reference data: 1.9**).
+
+
+### 2.4 Preparing OS
+
+-   Set the operating system locale. Add following line to the `/etc/environment` file.
+
+        LC_ALL=en_US.UTF-8
+
+-   If `/tmp` directory is mounted using `noexec` switch, the admin UI does not start, because it uses `/tmp` directory. 
+    Check is `/tmp` directory mounted using `noexec` switch:
+
+        mount | grep /tmp
+
+    Does the output contain `/tmp` and `noexec`?
+    
+        /dev/loop0 on /tmp type ext3 (rw,noexec,nosuid,nodev)
+
+    If yes, then `noexec` switch must be removed modifying `/etc/fstab` file. In addition, the directory must be mounted again to make the changes effective immediately:
+    
+        mount -o remount,exec /tmp
+
+
+### 2.5 Installation
+
+To install the X-Road security server software on *RHEL7* operating system, follow these steps.
+
+1.  Add X-Road package repository (**reference data: 1.1**) and Extra Packages for Enterprise Linux (EPEL) repository:
+
+        sudo yum install https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+        sudo yum-config-manager --add-repo https://artifactory.niis.org/xroad-release-rpm/rhel/7/current
```

Comment:
yum-config-manager is provided by package _yum-utils_, which might not be present in minimal installations.

Reply:
Added instructions for installing `yum-utils` in section "2.4 Preparing OS".